schema/release-schema.json

```
@@ -2079,6 +2079,15 @@
             "ZWR",
             null
           ]
+        },
+        "date": {
+          "title": "Date",
+          "description": "The date when the value is set. The value date is used when analysing historical values due to inflation over time.",
```

"when" -> "on which", for consistency with the descriptions of other date fields.

I think the usage example from the issue description is clearer:

```suggestion
          "description": "The date on which the value is set. The value date is useful to correctly sum values over longer periods of time, especially in contexts with high inflation.",
```